locales/fr.po

```diff
@@ -22,19 +22,19 @@ msgstr ""
 
 #: frontend/src/metabase/home/components/Activity.jsx:131
 msgid "Hello World!"
-msgstr "Bonjour tout le monde!"
+msgstr "Bonjour le Monde"
 
 #: frontend/src/metabase/home/components/Activity.jsx:132
 msgid "Metabase is up and running."
 msgstr "Metabase est opérationnel."
 
 #: frontend/src/metabase/home/components/Activity.jsx:176
 msgid "removed the filter {item.details.name}"
-msgstr "enlever le filtre {item.details.name}"
+msgstr "filtre {item.details.name} supprimé"
```

Comment:
As the file is named `Activity`, these strings are used to inform in a news feed what a user has done.
I propose rather _**a supprimé le filtre {item.details.name}**_ because at the end these strings will be like **_Jean_ a réalisé quelque chose de génial qu'il est difficile de décrire**

Ref :
https://github.com/metabase/metabase/blob/73c72782cb5078f1d23f6bd166983b4dc74b1327/frontend/src/metabase/home/components/Activity.jsx#L90-L94